### PR TITLE
Skip IBC address

### LINKED ISSUE #14: TODO: manage Escrow accounts
I find another edge case to manage: escrow accounts, i.e. 

https://www.mintscan.io/cosmos/account/cosmos1x54ltnyg88k0ejmk8ytwrhd3ltm84xehrnlslf (looks like to be for osmosis).

~5M atom on may 20th.
~9M atom on the airdrop file.

---

`cat snapshot_consolidated_10562840.json | jq '.[] | select(.address=="cosmos1x54ltnyg88k0ejmk8ytwrhd3ltm84xehrnlslf")'`

```json
{                            
  "address": "cosmos1x54ltnyg88k0ejmk8ytwrhd3ltm84xehrnlslf",                                                                
  "coins": [
    {
      "amount": "61010137",
      "denom": "ibc/12DA42304EE1CE96071F712AA4D58186AD11C3165C0DCDA71E017A54F3935E66"
    },
    {
      "amount": "10000000",   
      "denom": "ibc/1542F8DC70E7999691E991E1EDEB1B47E65E3A217B1649D347098EE48ACB580F"
    },
    {
      "amount": "2039990880534",
      "denom": "ibc/1BE91D67775723D3230A9A5AC54BB29B92A5A51B4B8F20BBA37DF1CFA602297C"
    },
   [........],
    {
      "amount": "9197744591079",
      "denom": "uatom"                                                                                                       
    } 
  ], 
  "vote": ""                
}              
```

---

```console
$ gaiad q ibc-transfer escrow-address transfer channel-141 --chain-id 1 --node https://rpc.cosmos.network  

cosmos1x54ltnyg88k0ejmk8ytwrhd3ltm84xehrnlslf
```

I suggest that we just skip them completely.

### PR DESCRIPTION
**Pull request by giansalex**
closes #14

### CODE REVIEW

File: consolidate/process_consolidated.go [hunk 1]
```diff
@@ -205,6 +212,11 @@ func convertAddress(cosmosAddress string) (string, error) {
 }
 
 func skip(address string) bool {
+	// skip ibc escrow address
+	if ibcEscrowAddress[address] {
+		return true
```
Comment (moul) [inline consolidate/process_consolidated.go:217]:
```suggestion
		// return true
```

our goal is not to skip them but to lock them

**Comment by moul** [comment]
Hey @giansalex, thank you for your contribution; we'll need it.

However, the goal now is not to skip the escrows but to list them so we can lock their balance until they prove that they can distribute them fairly to their users.

So please, just comment on the skip part, and keep the rest, especially the generated CSV file.

**Comment by giansalex** [comment]
@moul ok, considering that the ibc balance will be locked, is there any additional step to complete the final airdrop allocation?
cosmostation has been asked several times.

**Comment by moul** [comment]
To complete the final airdrop allocation: verifying because there are probably some errors.

To make the whole thing happen, then: prepare support for locking (technically and legally).

**Pull request merged**